src/main/java/gregtech/api/gui/widgets/TankWidget.java

```diff
@@ -102,9 +101,6 @@ public TankWidget setFluidRenderOffset(int fluidRenderOffset) {
     }
 
     public TankWidget setContainerClicking(boolean allowClickContainerFilling, boolean allowClickContainerEmptying) {
-        if (!(fluidTank instanceof IFluidHandler))
-            throw new IllegalStateException(
-                    "Container IO is only supported for fluid tanks that implement IFluidHandler");
```

Comment:
i do not know why this check was needed, as it's never required to be an IFluidHandler in the class. 
either way, it'll be handled properly in the mui2 port